CI/flatpak/com.obsproject.Studio.json

```diff
@@ -248,6 +248,16 @@
       "name": "obs",
       "buildsystem": "cmake-ninja",
       "builddir": true,
+      "secret-env": [
+        "RESTREAM_CLIENTID",
+        "RESTREAM_HASH",
+        "TWITCH_CLIENTID",
+        "TWITCH_HASH",
+        "YOUTUBE_CLIENTID",
+        "YOUTUBE_CLIENTID_HASH",
+        "YOUTUBE_SECRET",
+        "YOUTUBE_SECRET_HASH"
+      ],
```

Comment:
`secret-env` are only 'build-commands' and 'post-install'. In OBS cases, they are not needed.

Reply:
Of course, what did I have in mind when adding this :scream: